lib/geocoder/calculations.rb

```diff
@@ -42,7 +42,7 @@ module Calculations
     # Returns true if all given arguments are valid latitude/longitude values.
     #
     def coordinates_present?(*args)
-      args.all? { |a| a.is_a? Numeric and !a.to_f.nan? }
+      args.all? { |a| a.is_a? Numeric or !a.to_f.nan? }
```

Comment:
I highly doubt that this is the fix you are looking for. This change would make `coordinates_present?` return `true` on essentially any arguments, including e.g. `Float::NAN` (which is a numeric) and any string value (which becomes a non-NaN number when converted `to_f`).

```rb
[Float::NAN, 'foo bar'].all? { |a| a.is_a? Numeric or !a.to_f.nan? }
# => true
```